coord/src/info_command.c

```diff
@@ -49,9 +49,13 @@ static InfoFieldSpec toplevelSpecs_g[] = {
     {.name = "cleaning", .type = InfoField_WholeSum}};
 
 static InfoFieldSpec gcSpecs[] = {
-    {.name = "current_hz", .type = InfoField_DoubleAverage},
     {.name = "bytes_collected", .type = InfoField_WholeSum},
-    {.name = "effectiv_cycles_rate", .type = InfoField_DoubleAverage}};
```

Comment:
what were those fields?

Reply:
my guess are that those are legacy fields that we once had on the local FT.INFO command that at some point got removed.
I don't see anywhere on the code where those fields are being filled out by the RediSearch module.

Reply:
so they were always 0/NaN?

Reply:
idk about what values they held.

What I can see is that for the gc spec we use `replyKvArray` which checks to see if isSet was turned on for each field.
It the field wasn't set then it won't be sent.
So I think that explains why those fields weren't being shown to the end user.

Relevant code if the field was actually set:
`      
if (source->u.avg.count) {
        RedisModule_ReplyKV_Double(reply, key, source->u.avg.sum / source->u.avg.count);
} else {
        RedisModule_ReplyKV_Double(reply, key, 0);
}`